src/views/InteractiveMap/CardDialog.vue

```diff
@@ -14,7 +14,7 @@
               ></v-text-field>
             </v-flex>
             <v-flex xs12 md3>
-              <v-select
+              <AutoSelect
```

Comment:
Open to renaming suggestions. `v-select-auto-single`?

Reply:
`AutoVSelect`?